plugins/events-backend/src/service/http/HttpPostIngressEventPublisher.ts

```diff
@@ -45,6 +46,39 @@ class UnsupportedMediaTypeError extends CustomErrorBase {
   }
 }
 
+const defaultHttpDefaultBodyParser: HttpDefaultBodyParser = async (
```

Comment:
```suggestion
const defaultHttpBodyParser: HttpBodyParser = async (
```

I guess it is not a "default body parser", but a default implementation of a "body parser". However, maybe I miss something.